app/models/miq_report/generator.rb

```diff
@@ -88,6 +88,34 @@ def get_include
     include_as_hash(include.presence || invent_report_includes)
   end
 
+  def polymorphic_includes
+    @polymorphic_includes ||= begin
+      polymorphic_rels = []
+      top_level_rels   = []
+      top_level_rels  += Array(get_include.try(:keys))
```

Comment:
`get_include` could return a nil(maybe), symbol, array, or hash.

Do we think there could be a polymorphic at the second level?

Reply:
> Do we think there could be a polymorphic at the second level?

I decided to punt on this since it didn't seem like previously we were doing anything better in the deleted code in `Rbac::Filterer`... but yes, there probably could be.

Reply:
> `get_include` could return a nil(maybe), symbol, array, or hash.

Are you suggesting this needs to be smarter then?  Covers `nil` and `Hash` currently, but I think you are right in saying that it doesn't work with all of them (`Symbol` or `Array`).